plugins/nf-validation/build.gradle

```diff
@@ -54,7 +54,8 @@ dependencies {
     compileOnly "io.nextflow:nextflow:$nextflowVersion"
     compileOnly 'org.slf4j:slf4j-api:1.7.10'
     compileOnly 'org.pf4j:pf4j:3.4.1'
-    api 'com.github.everit-org.json-schema:org.everit.json.schema:1.14.1'
+    implementation 'org.json:json:20230227'
+    implementation 'dev.harrel:json-schema:1.5.0'
```

Comment:
What is the relevance of `implementation` vs `api` here?

Reply:
I'm not really sure but the docs from the library said to use `implementation` so I did